Rename Pico Probe -> Debug Probe (#3523)
raspberrypi/documentation: documentation/asciidoc/microcontrollers/debug-probe/getting-started.adoc

video::4RCZBZsfsek[youtube]

Connect the following:

* The Debug Probe "D" port to Pico H SWD JST-SH connector
* The Debug Probe "U" port, with the three-pin JST-SH connector to 0.1-inch header (male):
** Debug Probe `RX` connected to Pico H `TX` pin
** Debug Probe `TX` connected to Pico H `RX` pin
** Debug Probe `GND` connected to Pico H `GND` pin

> NOTE: If you have an "original" Raspberry Pi Pico or Pico W (without a JST-SH connector) you can still connect it to a Debug Probe. To do so, solder a male connector to the `SWCLK`, `SWDIO` and `GND` header pins on the board, and connect them to the Debug Probe "D" port using the alternate 3-pin JST-SH connector to 0.1-inch header (female) cable included with the Debug Probe.

Author: ```suggestion
NOTE: If you have a non-H Pico or Pico W (without a JST-SH connector) you can still connect it to a Debug Probe. Solder a male connector to the `SWCLK`, `GND`, and `SWDIO` header pins on the board. Using the alternate 3-pin JST-SH connector to 0.1-inch header (female) cable included with the Debug Probe, connect to the Debug Probe "D" port. Connect `SWCLK`, `GND`, and `SWDIO` on the Pico or Pico W to the `SC`, `GND`, and `SD` pins on the Debug Probe, respectively.
```
